server/tweetSearch.js

```diff
@@ -324,12 +324,22 @@ module.exports = function(client, fs, speakerFile) {
     }
 
     function addSpeaker(name) {
+        tweetUpdates.push({
```

Comment:
This looks pretty good; it seems like it'd make more sense (given the other update objects) to have the speaker operations come under one type with an object property listing whether they were added or removed; functionally it's the same either way though, so if you have any reason for having it this way then keep it pretty much